packages/owl-bot/src/core.ts

```diff
@@ -502,13 +502,24 @@ async function hasOwlBotLoop(
       per_page: 100,
     })
   ).data;
-  let count = 0;
-  for (const commit of commits) {
-    if (commit?.author?.login === OWLBOT_USER) count++;
-    else count = 0;
-    if (count >= circuitBreaker) return true;
+
+  // get the most recent commits (limit by circuit breaker)
+  const lastFewCommits = commits
+    .sort((a, b) => {
```

Comment:
@bcoe we may not need this sort, however I didn't see any documented guarantees `octokit.pulls.listCommits` returns sorted by date